index.html

```diff
@@ -205,6 +205,41 @@ <h2 class="lang">Skills</h2>
             </li>
         </ul>
     </section>
+    <section class="form-section">
+        <ul class="form-iner">
+            <li>
+              <h2 class="form-title">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            </li>
+            <li>
+              <form action="https://formspree.io/f/xyyoglkw" method="post" class="message-form">
+                <input type="text" name="name" placeholder="Name" maxlength="30" required class="form-control">
+                <input type="email" name="email" placeholder="Email" maxlength="30" required class="form-control">
+                <textarea name="message" placeholder="Enter message!" rows="5" maxlength="500" required class="form-control"></textarea>
+                <button type="submit" class="btn-form ">Get in touch</button>
+              </form>
+            </li>
+          </ul>
+    </section>
+    <footer class="footer-nav">
+        <ul class="icon-list">
+            <li>
+            <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i> </a>
+            </li>
+            <li>
+            <a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i> </a>
+            </li>
+            <li>
+            <a href="#"><i class="fa fa-medium" aria-hidden="true"></i> </a>
+            </li>
+            <li>
+            <a href="#"><i class="fa fa-github" aria-hidden="true"></i> </a>
+            </li>
+            <li>
+            <a href="#"><i class="fa fa-angellist" aria-hidden="true"></i> </a>
+            </li>
+        </ul>
+        <h3 class="email-title">tobinchilongojnr@gmail.com</h3>
```

Comment:
- [ ] [OPTIONAL]: To make your project even more professional, you should consider making this an `<a>` with the href attribute set your own email, this way, whenever the user clicks on this link, it will be redirected to a black email, with your email as recipient. You can read more about it [here](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Creating_hyperlinks#e-mail_links).